config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What do you think to export this in .env file and in this way all your configs will be accessible globally by `process.env.MyConfig`